pkg/node/node_address.go

```diff
@@ -305,6 +307,26 @@ func SetIPv6Router(ip net.IP) {
 	ipv6RouterAddress = ip
 }
 
+// GetIPv4HealthIP returns the IPv4 address of the local cilium-health endpoint
+func GetIPv4HealthIP() *net.IP {
+	return ipv4HealthAddress
+}
+
+// SetIPv4HealthIP sets the IPv4 address of the local cilium-health endpoint
+func SetIPv4HealthIP(ip *net.IP) {
+	ipv4HealthAddress = ip
+}
+
+// GetIPv6HealthIP returns the IPv6 address of the local cilium-health endpoint
+func GetIPv6HealthIP() *net.IP {
+	return ipv6HealthAddress
+}
+
+// SetIPv6HealthIP sets the IPv6 address of the local cilium-health endpoint
+func SetIPv6HealthIP(ip *net.IP) {
+	ipv6HealthAddress = ip
```

Comment:
why not just export these variables? that way these functions aren't needed.

Reply:
Just following the existing style.

Reply:
From discussion with @aanm , we want this convention as it allows us in the future to place mutexes around these objects in the library itself instead of placing the responsibility on the utilizer of these members.